<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>2340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMusic</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelBkg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>2340</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/img/bkg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>2170</y>
      <width>990</width>
      <height>6</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 200, 255);
background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="CButton" name="btnPlay">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>2200</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPlay.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnPause">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>2200</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPause.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnPlayPrev">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>2210</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPlayPrev.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnPlayNext">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>2210</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnPlayNext.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlbumName">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>1680</y>
      <width>200</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDuration">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>2110</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSingerName">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>1570</y>
      <width>600</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="CButton" name="btnOrder">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>2225</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnOrder.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnRandom">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>2225</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnRandom.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CLabel" name="labelPlayProgress" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>2158</y>
      <width>990</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text" stdset="0">
     <string/>
    </property>
    <property name="wordWrap" stdset="0">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlayingfile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>1495</y>
      <width>1000</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="CButton" name="btnSetting">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>2225</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnSetting.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="wdgSingerImg" native="true">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>1563</y>
      <width>627</width>
      <height>576</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/defaultSingerImg.jpg);
border-image-repeat: stretched</string>
    </property>
    <widget class="QFrame" name="frameSingerImg">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>391</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="labelSingerImg">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>211</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frameDemand">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>840</width>
      <height>170</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="CButton" name="btnDemandAlbum">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/album.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandAlbumItem">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/albumitem.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandPlaylist">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/playlist.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandSinger">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/singerdefault.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="CButton" name="btnDemandPlayItem">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/img/playitem.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>130</width>
       <height>130</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frameDemandLanguage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>990</width>
       <height>60</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="CLabel" name="labelDemandCN">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>110</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>  国语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandHK">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>110</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>  粤语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandKR">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>110</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>  韩语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandJP">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>110</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>  日语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandTAI">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>0</y>
        <width>110</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>  泰语</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandEN">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>0</y>
        <width>110</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>  英文</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="CLabel" name="labelDemandEUR">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>0</y>
        <width>140</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string>  小语种</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>520</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>X Music</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogoCompany">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>281</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>@MusicroSoft</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogoTip">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>491</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="CButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/btnExit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>labelBkg</zorder>
   <zorder>progressBar</zorder>
   <zorder>btnPlay</zorder>
   <zorder>btnPause</zorder>
   <zorder>btnPlayPrev</zorder>
   <zorder>btnPlayNext</zorder>
   <zorder>labelAlbumName</zorder>
   <zorder>labelDuration</zorder>
   <zorder>btnOrder</zorder>
   <zorder>btnRandom</zorder>
   <zorder>labelPlayProgress</zorder>
   <zorder>btnSetting</zorder>
   <zorder>wdgSingerImg</zorder>
   <zorder>labelSingerName</zorder>
   <zorder>frameDemand</zorder>
   <zorder>labelLogo</zorder>
   <zorder>labelLogoCompany</zorder>
   <zorder>labelLogoTip</zorder>
   <zorder>btnExit</zorder>
   <zorder>labelPlayingfile</zorder>
  </widget>
  <action name="action">
   <property name="text">
    <string>专辑</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CLabel</class>
   <extends>QLabel</extends>
   <header location="global">label.h</header>
  </customwidget>
    
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header location="global">ProgressBar.h</header>
  </customwidget>
  
  <customwidget>
   <class>CButton</class>
   <extends>QPushButton</extends>
   <header location="global">button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
